### PR TITLE
Update healthcheck and packages

### PR DESCRIPTION
**Pull request by KostinP**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/249

### CODE REVIEW

File: src/NewsService.Mappers/ResponsesMappers/NewsResponseMapper.cs [hunk 1]
```diff
@@ -31,8 +32,8 @@ public NewsResponse Map(DbNews dbNews)
                 throw new BadRequestException();
             }
 
-            User author = new User();
-            User sender = new User();
+            User author = new();
+            User sender = new();
```
Comment (MariyaSin) [inline src/NewsService.Mappers/ResponsesMappers/NewsResponseMapper.cs:37]: Why did you remove "User"?

File: src/NewsService.Business/EditNewsCommand.cs [hunk 2]
```diff
@@ -4,33 +4,34 @@
...
-        private readonly IValidator<News> validator;
+        private readonly INewsRepository _repository;
+        private readonly INewsMapper _mapper;
+        private readonly INewsValidator _validator;
 
         public EditNewsCommand(
             [FromServices] INewsRepository repository,
             [FromServices] INewsMapper mapper,
-            [FromServices] IValidator<News> validator)
+            [FromServices] INewsValidator validator)
```
Comment (djino97) [inline src/NewsService.Business/EditNewsCommand.cs:21]: remove FromServices

File: src/NewsService.Mappers/ResponsesMappers/NewsResponseMapper.cs (hunk 1 above)
Comment (KostinP) [reply src/NewsService.Mappers/ResponsesMappers/NewsResponseMapper.cs:37]: Why not?

**Review by djino97 — APPROVED** [review]

**Review by 24eeeee — APPROVED** [review]

**Review by TIove — APPROVED** [review]

File: src/NewsService.Business/CreateNewsCommand.cs [hunk 3]
```diff
@@ -4,34 +4,36 @@
...
-        private readonly IValidator<News> validator;
+        private readonly INewsRepository _repository;
+        private readonly INewsMapper _mapper;
+        private readonly INewsValidator _validator;
 
         public CreateNewsCommand(
             [FromServices] INewsRepository repository,
             [FromServices] INewsMapper mapper,
-            [FromServices] IValidator<News> validator)
+            [FromServices] INewsValidator validator)
```
Comment (BelehovEgor) [inline src/NewsService.Business/CreateNewsCommand.cs:23]: similarly

**Review by BelehovEgor — APPROVED** [review]
